### PR TITLE
kv secrets mount case-sensitivity note

### PR DESCRIPTION
**Pull request by zalary**

### CODE REVIEW

File: website/pages/docs/secrets/kv/index.mdx [hunk 1]
```diff
@@ -13,6 +13,8 @@ modes. It can be a generic Key-Value store that stores one value for a key.
 Versioning can be enabled and a configurable number of versions for each key
 will be stored.
 
+Note that `kv` secrets mounts are case-sensitive.
```
Comment (ncabatoff) [inline website/pages/docs/secrets/kv/index.mdx:16]: What does this mean?  That we preserve case for keys?

Comment (yhyakuna) [reply website/pages/docs/secrets/kv/index.mdx:16]: > What does this mean? That we preserve case for keys?

@ncabatoff  apparently so?  https://app.asana.com/0/112957393038545/1109620483178716/f

Comment (mikegreen) [inline website/pages/docs/secrets/kv/index.mdx:16]: To comply with the use of "note" in other places, could we use this syntax to call it out in a box? 
```suggestion
~> **NOTE:** 
```

**Comment by aphorise** [comment]
I believe this request may still be relevant particularly since there are confusions regarding the default case sensitivity that's expected from most systems.

@yhyakuna @brianshumate please advice.

**Comment by yhyakuna** [comment]
Since this PR was originally opened, the folder structure have changed. I opened a [PR 16473](https://github.com/hashicorp/vault/pull/16743), so closing this.

**Pull request closed without merging**